.circleci/config.yml

```diff
@@ -18,18 +72,15 @@ jobs:
              curl "$SDK_BASE_URL/$SDK_FILE" -sS -o "$SDK_FILE"
              sha256sum -c --ignore-missing sha256sums
 
-      - checkout:
-          path: ~/openwrt_packages
-
       - run:
           name: Prepare build_dir
           working_directory: ~/build_dir
           command: |
              tar Jxf ~/sdk/$SDK_FILE --strip=1
              cat > feeds.conf <<EOF
-             src-git base https://github.com/openwrt/openwrt.git
+             src-git base https://github.com/openwrt/openwrt.git;$BRANCH
              src-link packages $HOME/openwrt_packages
-             src-git luci https://github.com/openwrt/luci.git
+             src-git luci https://github.com/openwrt/luci.git;$BRANCH
```

Comment:
good catch !!

Reply:
Yes, tested both master and release branches